### ISSUE TITLE
3D: add a compass or similar

### ISSUE THREAD

**Issue by danduk82** [geoadmin/mf-geoadmin3#4504]
As a : User
I want to: know my orientation using pegman or flight mode (compass?)
because: I am lost

**Comment by davidoesch** [comment]
80% of the user cannot read a compass ( 270° is which direction) -but relative positioning and orientation is done by : placing names iswhere user is helped

What I can imagine: that we place the bussoles left and right of the help text

**Comment by procrastinatio** [comment]
```
       |
       |
330    -
       |  <
       |
300    -
       |
                
                      v   
       ------|--------------|-------------|-
            90             180           270
```

**Comment by davidoesch** [comment]
Yeah flight Simulator. For this there is a complete cesium integration.

**Comment by davidoesch** [comment]
More important: performance needs to be fixed... Really

**Comment by boecklic** [comment]
@boecklic commented: 80% of the user cannot read a compass ( 270° is which direction) -but relative positioning and orientation is done by : placing names iswhere user is helped

What I can imagine: that we place the bussoles left and right of the help text

**Comment by boecklic** [comment]
@boecklic commented: 
```
       |
       |
330    -
       |  <
       |
300    -
       |
                
                      v   
       ------|--------------|-------------|-
            90             180           270
```

**Comment by boecklic** [comment]
@boecklic commented: Yeah flight Simulator. For this there is a complete cesium integration.

**Comment by boecklic** [comment]
@boecklic commented: More important: performance needs to be fixed... Really
